.github/settings.yml

```diff
@@ -12,7 +12,9 @@ repository:
   allow_squash_merge: true
   allow_merge_commit: false
   allow_rebase_merge: false
+  allow_auto_merge: true
   delete_branch_on_merge: true
+  use_squash_pr_title_as_default: true
```

Comment:
:question: did you find these or are you trying them out?
I'm not seeing them on https://github.com/apps/settings, but they look reasonable

Reply:
Yeah, it turns out the settings app is just transforming it's yaml into the JSON used in the GH API, so I'm taking a flyer that those fields exist and will get applied.